### PR TITLE
Feature/error handler

### PR DESCRIPTION
**Pull request by tema1411**
added error handler for backend api

### CODE REVIEW

**Comment by tema1411** [comment]
Not sure is right folder for httpStattusCode.helper.ts file

File: server/src/helpers/errorHandler.helper/httpStattusCode.helper.ts [hunk 1]
```diff
@@ -0,0 +1,380 @@
+/**
```
Comment (taras-dubyk) [inline server/src/helpers/errorHandler.helper/httpStattusCode.helper.ts:1]: Should be in "constants" folder

File: server/src/helpers/errorHandler.helper/errorHandler.helper.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import HttpStatusCode from './httpStattusCode.helper';
+
+export class ErrorHandler extends Error {
```
Comment (taras-dubyk) [inline server/src/helpers/errorHandler.helper/errorHandler.helper.ts:4]: Can "ErrorHandler" class be renamed to something like "CustomError"?

Comment (tema1411) [reply server/src/helpers/errorHandler.helper/errorHandler.helper.ts:4]: I think it's better to leave it as it is. The developer will start typing Error and the IDE will help him.  No need to remember what it was named to start typing

Comment (tema1411) [reply server/src/helpers/errorHandler.helper/errorHandler.helper.ts:4]: @taras-dubyk May be ErorResponse?

Comment (taras-dubyk) [reply server/src/helpers/errorHandler.helper/errorHandler.helper.ts:4]: Let it be, leave ErrorHandler for now

**Pull request merged**